project1/styles.css

```diff
@@ -0,0 +1,123 @@
+body {
+    background-color: hsl(0, 0%, 96%);
+}
+
+header {
+    width: 80%;
+    margin: auto;
+}
+
+main {
+    margin-left: auto;
+    margin-right: auto;
+    width: 60%;
+    line-height: 1.8;
+}
+
+#octopicture {
+    width: 70%;
+    margin: auto;
+}
+
+figure {
+    margin: auto;
+    width: 60%;
+}
+
+img {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 100%;
+}
+
+h1 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 62px;
+    font-weight: bolder;
+    color: hsl(177, 32%, 51%);
+    text-align: left;
+}
+
+h2 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 32px;
+    font-weight: bold;
+    color: hsl(0, 0%, 20%);
+}
+
+h3 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 18px;
+    font-weight: normal;
+    color: hsl(0, 0%, 20%);
+}
+
+h4 {
+    font-family: 'Roboto Slab', serif;
+    font-size: 18px;
+    font-weight: normal;
+    color: hsl(0, 0%, 20%);
+}
+
+p {
+    font-family: 'Roboto', regular;
+    font-size: 18px;
+    font-style: normal;
+    color: hsl(0, 0%, 20%);
+    text-align: left;
+    width: auto;
+
+}
+
+q {
+    font-family: 'Roboto', regular;
+    font-size: 22px;
+    font-style: normal;
+    color: rgb(255, 255, 255);
+    text-align: center;
+    width: auto;
+}
+
+/*#video {}*/
+
+/*#wrapper {}*/
+
+iframe {
+    display: block;
+    /*float: left;*/
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 20px;
+
+}
+
+#pullquote {
+    margin-top: 10px;
+    margin: auto;
```

Comment:
This is actually overriding your declaration on line 96, since `margin` alone sets _all_ margins to whatever value you give it. You can see this in developer tools because the `margin-top` declaration is crossed out:

<img width="263" alt="screen shot 2019-02-16 at 5 34 06 pm" src="https://user-images.githubusercontent.com/2653980/52905994-2f3efa00-3211-11e9-89f6-9631e9b20b32.png">

If you delete line 96 and update line 97 like so:

```suggestion
    margin: 10px auto 0;
```

Your pullquotes will have a margin top of 10px, a left and right margin of auto and no margin on the bottom.